pillar/params.sls

```diff
@@ -75,3 +75,7 @@ proxy:
 reboot:
   group:          'default'
   directory:      'opensuse.org/rebootmgr/locks'
+
+transactional-update:
+  timer:
+    on_calendar: '*-*-* 00/12:00:00' # See https://www.freedesktop.org/software/systemd/man/systemd.time.html#Calendar%20Events for syntax
```

Comment:
_TODO_: it seems we could be creating too many snapshots and downloading too many times the same packages if we lower the frequency for this timer. @thkukuk please take a looks at [this log](https://gist.github.com/inercia/4592aff3651da259d2ea512339cf5f08)... is this a known behaviour in `Build22.17`?